File: backend/src/main/java/com/wildBirds/BlueChat/api/rest/dto/UserDto.java
package com.wildBirds.BlueChat.api.rest.dto;

import lombok.*;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class UserDto {

    private Long idUser;
    private String nick;
    private String email;
    private String description;
    private PhotoDto photoDto;



}

File: mobile/src/app/repository/user/userDto.ts
import {PhotoDto} from '../photo/photoDto';

export class UserDto {
    public idUser : number;
    public nick: string;
    public email: string;
    public description: string;
    public photoDto : PhotoDto;

    constructor(id: number = 0, name: string = "", email: string = "", description: string = "", photo: PhotoDto = null) {
        this.idUser = id;
        this.nick = name;
        this.email = email;
        this.description = description;
        this.photoDto = photo;
    }

}